src/components/_common/office-details/OfficeDetails.tsx

```diff
@@ -0,0 +1,50 @@
+import { Heading } from '@navikt/ds-react';
+import { PlaceFilled } from '@navikt/ds-icons';
+import classNames from 'classnames';
+import { translator } from 'translations';
+import { normalizeReceptionAsArray } from './utils';
+import { Reception } from './reception/Reception';
+import { OfficeDetailsData } from 'types/content-props/office-details-props';
+import { usePageConfig } from 'store/hooks/usePageConfig';
+import { PhonePoster } from './phonePoster/PhonePoster';
+import { OfficeInformation } from './officeInformation/OfficeInformation';
+
+import styles from './OfficeDetails.module.scss';
+export interface OfficeDetailsProps {
+    officeData: OfficeDetailsData;
+}
+
+export const OfficeDetails = ({ officeData }: OfficeDetailsProps) => {
+    const { language } = usePageConfig();
+    const { brukerkontakt } = officeData;
+    const getOfficeTranslations = translator('office', language);
+
+    const publikumsmottak = normalizeReceptionAsArray(
+        brukerkontakt.publikumsmottak
+    );
+
+    return (
+        <div className={styles.wide}>
+            <div className="content">
```

Comment:
Hvor kommer `content`-klassen fra? Kan denne defineres i en module kanskje?

Trenger vi 3 nivåer med div'er her?

Reply:
Ja, vi kommer unna med to nivåer ser jeg...